src/_cffi_src/openssl/ssl.py

```diff
@@ -609,6 +616,16 @@
 static const long Cryptography_HAS_ALPN = 1;
 #endif
 
+/* SSL_CTX_set_cert_cb was added in OpenSSL 1.0.2. */
+#if OPENSSL_VERSION_NUMBER < 0x10002001L && !defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Looks like Libre doesn't have this so you'll want to do `&& defined` rather than `&& !defined`